cli/src/cli_util.rs

```diff
@@ -413,11 +413,23 @@ impl CommandHelper {
     /// Loads workspace and repo, then snapshots the working copy if allowed.
     #[instrument(skip(self, ui))]
     pub fn workspace_helper(&self, ui: &Ui) -> Result<WorkspaceCommandHelper, CommandError> {
-        Ok(self.workspace_helper_with_stats(ui)?.0)
+        let (workspace_command, stats) = self.workspace_helper_with_stats(ui)?;
+        print_snapshot_stats(
+            ui,
+            &stats,
+            workspace_command.env().path_converter(),
+            SnapshotContext::Automatic,
+        )?;
+        Ok(workspace_command)
     }
 
     /// Loads workspace and repo, then snapshots the working copy if allowed and
     /// returns the SnapshotStats.
+    ///
+    /// Note that unless you have a good reason to do so, you should always call
```

Comment:
nit: missing a "not" somewhere (e.g. "unless you have a good reason not to")

Reply:
Thanks, fixed :)